SalesforceCommerceCloud/pwa-kit
# Replace some translations with full sentences

On the Checkout's Shipping page, some translations were using variables that we set to an English text. As a result, there will be a mix of the target language and English in the same sentence.

This PR fixes that by replacing those translations with full sentences instead. For example:

Before: `defaultMessage: 'Please select your {stateOrProvince}.'`
After: `defaultMessage: 'Please select your state.'`

Related PR: #1732 

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

1. `npm start` to run a local website
2. Switch to a different locale
3. Add an item to your basket
4. Go to Checkout
5. Go to the Shipping step
6. Don't fill out the form, and then try to submit it
7. There will be error messages on the blank form fields
8. Verify that those errors do not have English words (e.g. "state", "zip code")

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
